python/setup.in.py

```diff
@@ -55,7 +55,7 @@ def finalize_options(self):
 
 setup(
     name='openvino_tensorflow',
-    version='2.0.0',
+    version='2.1.0',
```

Comment:
OVTF version also needs to be updated in openvino_tensorflow/version.cc

Reply:
done